torch/_dynamo/config.py

```diff
@@ -43,7 +43,7 @@
 cache_size_limit = 64
 
 # specializing int/float by default
-specialize_int_float = True
+specialize_int_float = False
```

Comment:
for my understanding - why is this default False? Is this config a no-op unless `dynamic_shapes=True` is set?

Reply:
So the idea is post Avik's PR, `specialize_int_float` does nothing unless dynamic_shapes=True is set, and we don't want to specialize on int inputs (see for example https://github.com/pytorch/pytorch/issues/93468#issuecomment-1442732186 of overspecialization even with dynamic=True). But export might want to specialize, so we let them turn ON specialization if they want.